tests/replicaset/writeconcernerror-002.phpt

```diff
@@ -23,7 +23,6 @@ try {
     $retval = $manager->executeBulkWrite(NS, $bulk, $w);
 } catch(MongoDB\Driver\Exception\BulkWriteException $e) {
     printWriteResult($e->getWriteResult(), false);
-    $e->getWriteResult(true);
```

Comment:
I didn't see the point of this method call, as we're already calling it on the line above. Admittedly, this change should be in its own commit rather than lumped into "Rename WriteConcernError tests" (I can break it up if you like).

AFAIK, this was only testing that zpp expects no arguments, which didn't seem relevant for a WriteConcernError test.